com.googlecode.cppcheclipse.core/src/com/googlecode/cppcheclipse/core/Checker.java

```diff
@@ -74,7 +74,8 @@ public Checker(IConsole console, IPreferenceStore projectPreferences,
 		}
 		
 		String binaryPath = CppcheclipsePlugin.getConfigurationPreferenceStore()
-		.getString(IPreferenceConstants.P_BINARY_PATH);
+		.getString(IPreferenceConstants.P_BINARY_PATH)
```

Comment:
Instead of doing the placeholder replacement each time in the code please rather come up with a util method and use this method everywhere where the binary path is necessary.

Reply:
Yes, I wanted to do that primarily but wanted to modify as little as possible of your code. Making up whole util class just to get the PR reject on the FileField -> StringField grounds would be waste of time for nothing.